packages/demand-bloc-payment/src/PaymentBloc.ts

```diff
@@ -75,7 +75,7 @@ export class PaymentBloc {
     this.pendingInitialisation = getCancellablePromise(
       Promise.all([
         this.provider.initialize(payer),
-        paymentCardsEnabled && payer ? this.provider.getSavedCards(payer) : null,
+        paymentCardsEnabled ? this.provider.getSavedCards() : null,
```

Comment:
do we need a payer in the line above when initializing provider?

Reply:
well I found that Adyen lib is using that for some `shopperData` and since I don't know what that is I decided to leave that as it is 

https://github.com/karhoo/web-lib-demand/blob/master/packages/demand-bloc-payment/src/providers/AdyenProvider.ts#L106-L109